test.js

```diff
@@ -1,57 +1,57 @@
+import {Writable as WritableStream} from 'stream';
 import test from 'ava';
-import hookStd from 'hook-std';
 import Ora from './';
 
 const spinnerChar = process.platform === 'win32' ? '-' : '⠋';
 
-function readOutput(spinner, callback) {
-	let out = '';
-
-	const unhook = hookStd.stderr({silent: true}, output => {
-		out += output;
+test.beforeEach(t => {
+	const noop = () => {};
+	const stream = new WritableStream({
+		write(chunk, encoding, callback) {
+			t.context.output += chunk.toString();
```

Comment:
@josmardias It's not gone, just collapsed: https://github.com/sindresorhus/ora/pull/18#discussion_r70980660

To answer you questions. If a multi codepoint unicode character is split between write calls you'll `toString()` encode only part of the character, causing it to be an invalid character. That's why you should never call `.toString()` there, but rather at the end when you use `output`. Alternatively you can [set the encoding of the stream](https://nodejs.org/api/stream.html#stream_readable_setencoding_encoding).

> Here for example, is doing something very similar:

That implementation is bad. Use [`get-stream`](https://github.com/sindresorhus/get-stream) instead for when you need a string from a stream.